updatecli/updatecli.d/golang.yaml

```diff
@@ -60,8 +60,10 @@ targets:
     kind: file
     spec:
       file: ./Jenkinsfile_k8s
-      matchPattern: 'golang:.*'
-      replacePattern: 'golang:{{ source `latestGoVersion` }}'
+      matchPattern: >-
+        'golang:.*'
+      replacePattern: >-
+        'golang:{{ source `latestGoVersion` }}'
```

Comment:
Why with `>-` instead of the same line?

Reply:
Was there a problem of regexp parsing?

Reply:
yep, the single quote escaping broke my head, so I went to the block litteral.

Reply:
(e.g. I'm lazy :D )